lumeer-storage/lumeer-storage-api/src/main/java/io/lumeer/storage/api/filter/CollectionAttributeFilter.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.lumeer.storage.api.filter;
+
+import io.lumeer.api.model.ConditionType;
+
+public class CollectionAttributeFilter extends AttributeFilter {
```

Comment:
I'm a little bit confused here. I can see that we have two classes called `CollectionAttributeFilter` and `LinkAttributeFilter`.

Reply:
One is for `search` filter purpose and the other is for rest & db purpose. I will rename them to make it clear what it does.